### ISSUE TITLE
Methods for preparing samples

### ISSUE THREAD

**Issue by DeirdreLoughnan** [temporalecologylab/TreeRings#3]
I met with someone from the Daniels lab yesterday to discuss the sanding process. Unfortunately we do need to have an official safety orientation before we can use the room. I am coordinating that for next week for both Dajeong and I, and then we can learn how the belt sander works! 

But in preparation of the sanding, I will also need to purchase some ppe. There are safety glasses we can borrow, but we should bring our own respirators. 

That said, @FrederikBaumgarten you mentioned finding a recent paper that proposed using a microtome to prepare surfaces for tree ring imaging. Could you share a link to that publication here? I am curious to read their proposed procedure and whether our samples being dry would be an issue.

**Comment by FrederikBaumgarten** [comment]
Here is the paper:
The core-microtome: A new tool for surface preparation on cores and time series analysis of varying cell parameters
https://doi.org/10.1016/j.dendro.2009.09.002

Best is fresh, but most of the cores are cut after wetting the surface with a little water

**Comment by lizzieinvancouver** [comment]
@DeirdreLoughnan @FrederikBaumgarten Just chiming in here to chime in that I picked up more cookies and will return with them in later November (actually I picked up what I would call logs and with much help, got them all cut in a jiffy).

**Comment by DeirdreLoughnan** [comment]
We had a great discussion at lab meeting this week about using the microtome. Unfortunately given the size of our cookies (and presumably the larger ones Lizzie is brining back), the microtome we have in the lab has too small a blade for us to use. 

So we will move forward with the previous plan to just them.

**Comment by DeirdreLoughnan** [comment]
@afong3 That is a good question. I think a minimum thickness would be about 2-3 cm and max 10 cm. The really large cookies tend to be thicker.

I think while sanding the cookies we can try and make them as parallel as possible. But this might take some practice first. 

Do Rasp pi cameras have auto focus? How difficult would this be to include?

**Comment by lizzieinvancouver** [comment]
@afong3 Agree with @DeirdreLoughnan but will add the idea that we build something (some sort of holder) that allows us to level the samples.

**Comment by afong3** [comment]
I think for sample preparation it would be best if I had access to the belt sander. I talked with @DeirdreLoughnan about the hoops we need to jump through to get access and we think it may be best to get our own belt sander ([~$300](https://www.homedepot.ca/product/ryobi-120v-4-inch-by-36-inch-belt-disc-sander/1000747387)) and a marble slab for honing in a parallel plane for a finishing stage. Is [this ](https://www.homedepot.ca/product/fox-run-marble-pastry-board-white-16x20-inches/1001390831) what you were imagining @DeirdreLoughnan?

What do you think? @lizzieinvancouver

**Comment by DeirdreLoughnan** [comment]
@afong3 yes, we could just buy a surface plate, but I agree any piece of smooth marble or granite (granite tiles perhaps) would work just fine.

**Comment by lizzieinvancouver** [comment]
@afong3 and @DeirdreLoughnan Thanks for working on this! We can definitely buy sanding stuff we need (tell me if the total goes above $1000) so please do (assuming we have space for it in the lab, but we can find other space if needed)!
